bookmarks/api/routes.py

```diff
@@ -88,6 +89,16 @@ def get_serializer_context(self):
         return {'user': self.request.user}
 
 
+class VersionViewSet(viewsets.ViewSet,
+                     mixins.ListModelMixin):
+
+    def list(self, request):
```

Comment:
I'm not an expert on Django REST Framework, but will this need to change if this ViewSet could potentially hold other endpoints?